pkg/eventemitter/eventemitter.go

```diff
@@ -65,33 +66,51 @@ func (ee eventEmitter) EmitEvent(object runtime.Object, eventType, reason, msg s
 }
 
 func (ee eventEmitter) EmitProgressingForConfig() {
-	config := ee.getConfigForEmitter()
+	config, err := ee.getConfigForEmitter()
+	if err != nil {
+		log.Printf(err.Error())
+		return
+	}
+
 	ee.EmitEvent(config, corev1.EventTypeNormal, ProgressingReason, ProgressingMessage)
 }
 
 func (ee eventEmitter) EmitFailingForConfig(reason, message string) {
-	config := ee.getConfigForEmitter()
+	config, err := ee.getConfigForEmitter()
+	if err != nil {
+		log.Printf(err.Error())
+		return
+	}
+
 	ee.EmitEvent(config, corev1.EventTypeWarning, fmt.Sprintf("%s: %s", FailedReason, reason), fmt.Sprintf("%s: %s", FailedMessage, message))
 }
 
 func (ee eventEmitter) EmitAvailableForConfig() {
-	config := ee.getConfigForEmitter()
+	config, err := ee.getConfigForEmitter()
+	if err != nil {
+		log.Printf(err.Error())
```

Comment:
ditto, add a specific message to `Available`

Reply:
Done.